proto/buf.yaml

```
@@ -0,0 +1,25 @@
+# Generated by "buf config migrate-v1beta1". Edit as necessary, and
+# remove this comment when you're finished.
+#
+# This module represents the "proto" root found in
+# the previous configuration.
+version: v1
+name: buf.build/regen/regen-ledger
```

```suggestion
version: v1
name: buf.build/umee-network/umee
```
Note 100% if we use github repository name here, or something else